docs/documentation/make-first-prototype/add-questions.md

```diff
@@ -0,0 +1,52 @@
+# Add questions to your question pages
+
+Copy the code from the [GOV.UK Design System](https://design-system.service.gov.uk/) to add page elements like radios and text inputs.
+
+## Add radios to question 1
+
+1. Go to the [stacked radios section of the Design System](https://design-system.service.gov.uk/components/radios/#stacked-radios).
+2. Select **HTML**, then select **Copy code**.
+3. Open `juggling-balls.html` in your `app/views` folder.
+4. Replace the 2 example `<p>...</p>` paragraphs with the HTML code you copied.
+5. Delete the whole `<h1>` tag that has the text “Heading or question goes here” - it should be line 16.
+5. In your copied code, change the text in the `<h1>` tag from `Where do you live?` to `How many balls can you juggle?`.
```

Comment:
```suggestion
6. In your copied code, change the text in the `<h1>` tag from `Where do you live?` to `How many balls can you juggle?`.
```

Reply:
Thanks Ollie - fixed in https://github.com/alphagov/govuk-prototype-kit/pull/805/commits/871d36b80c663194857e7a0708d31a7b6e213ff2